hasher-matcher-actioner/src/OpenMediaMatch/blueprints/matching.py

```diff
@@ -168,6 +249,59 @@ def query_index(
     return results
 
 
+def query_index_threshold(
+    signal: str, signal_type_name: str, threshold: int
+) -> t.Sequence[IndexMatchUntyped[SignalSimilarityInfo, int]]:
+    storage = get_storage()
+    signal_type = _validate_and_transform_signal_type(signal_type_name, storage)
+
+    try:
+        signal = signal_type.validate_signal_str(signal)
+    except Exception as e:
+        abort(400, f"invalid signal: {e}")
+
+    index = _get_index(signal_type)
+
+    if index is None:
+        abort(503, "index not yet ready")
+    
+    current_app.logger.debug("[lookup_signal_threshold] querying index")
```

Comment:
Probably put this log statement after the check that the index implements query_threshold